src/engine.rs

```diff
@@ -63,6 +67,8 @@ pub trait Pool: Debug {
                          -> EngineResult<()>;
     fn add_blockdev(&mut self, path: &Path) -> EngineResult<()>;
     fn add_cachedev(&mut self, path: &Path) -> EngineResult<()>;
+    fn remove_blockdev(&mut self, path: &Path) -> EngineResult<()>;
```

Comment:
I'm going to take these Path arguments as being a temporary hack, like the fact that get_id() is based on the Path.

I think that we will have to figure out something far better to pass to the engine methods, but I don't know what that is.